code/modules/mob/living/living_defense.dm

```diff
@@ -424,3 +424,14 @@
 		return TRUE
 	else
 		return FALSE
+
+/mob/living/proc/sethellbound()
+	if(mind)
+		mind.hellbound = TRUE
+		return TRUE
+	return FALSE 
+
+/mob/living/proc/hellbound()
```

Comment:
I think this would make more sense as is_hellbound or something since hellbound() could seem like the verb rather than is